components/MDX/UpcomingEvents.tsx

```diff
@@ -0,0 +1,36 @@
+import { getTranslations } from 'next-intl/server';
+import type { FC } from 'react';
+
+import BlogPostCard from '@/components/Common/BlogPostCard';
+import getBlogData from '@/next-data/blogData';
+
+const UpcomingEvents: FC = async () => {
+  const t = await getTranslations();
+  const { posts } = await getBlogData('events', 0);
+
+  const currentDate = new Date();
+  const filteredPosts = posts.filter(post => post.date > currentDate);
+
+  const fallbackPosts = Array(2).fill({
+    title: t('components.mdx.upcomingEvents.defaultTitle'),
+    categories: ['events'],
+  });
+
+  const mappedPosts = fallbackPosts.map((post, key) => {
+    const actualPost = filteredPosts[key] || post;
+
+    return (
+      <BlogPostCard
+        key={actualPost.slug || key}
+        title={actualPost.title}
+        category={actualPost.categories[0]}
+        date={actualPost.date}
+        slug={actualPost.slug}
+      />
+    );
+  });
+
+  return <div className="flex flex-row gap-3 xs:flex-col">{mappedPosts}</div>;
```

Comment:
```suggestion
  return <div className="flex flex-row gap-3 md:flex-col">{mappedPosts}</div>;
```

it would be necessary to switch to `md` to avoid this "crushed" effect

<img width="842" alt="Capture d’écran 2024-01-26 à 09 14 06" src="https://github.com/nodejs/nodejs.org/assets/97875033/8ab1812c-bb7e-4e7b-b0be-bea1ae642567">